apps/mobile/package.json

```diff
@@ -62,6 +63,7 @@
     "@stacks/common": "6.13.0",
     "@stacks/transactions": "6.15.0",
     "@stacks/wallet-sdk": "6.15.0",
+    "@tanstack/query-core": "5.59.13",
```

Comment:
Is this replacing `@tanstack/react-query` here? Should we remove that or are they for different use cases

Reply:
Our services layer might rely on `QueryClient`, but we don't want it to be react-specific and undernearth the react impl is query core, so think we'll need this. Not here for now though so can remove

Reply:
OK thanks, I'm just curious 👍